applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -47,15 +47,15 @@ def __init__(self, main_model_part, custom_settings):
 
         # Setting minimum buffer
         scheme_type = self.dynamic_settings["scheme_type"].GetString()
-        if("bdf" in scheme_type):
-            order = int(scheme_type.replace("bdf",""))
+        if("bdf" in scheme_type or scheme_type == "backward_euler"):
```

Comment:
I would use `if(scheme_type.startswith("bdf") or scheme_type == "backward_euler"):`
I think this is clearer

otherwise I could give the following string and it would still work: `String_bdf_to_fool_vicente`

Reply:
In this case you are right

Reply:
+1 for @philbucher